src/lib/RichText/Converter/Render/Embed.php

```diff
@@ -181,6 +181,7 @@ protected function extractLinkParameters(DOMElement $embed): ?array
         $title = $link->getAttribute('xlink:title');
         $id = $link->getAttribute('xml:id');
         $class = $link->getAttribute('ezxhtml:class');
+        $siteAccess = $link->getAttribute('xlink:siteaccess');
```

Comment:
SonarCloud reports here unused `$siteAccess` variable.

Reply:
Indeed, shouldn't be there